ansible/group_vars/common.yml

```diff
@@ -78,3 +78,8 @@ delfin_release_version: v1.2.1
 #################
 # URLs, Environment Variables, IP addresses and Ports list
 soda_delfin_url: "http://{{ host_ip }}:8190"
+
+# Inform the SODA Dashboard whether SODA Multicloud UI should support only service plans or backends
```

Comment:
Please mark it as Fixme. This should be extended across SODA projects and not limited to multicloud. Or there can be a altogether new approach to handle this